river/cluster/clustream.py

```diff
@@ -305,13 +305,15 @@ def __init__(
         if x is not None and sample_weight is not None:
             # Initialize with sample x
             self.n_samples = 1
-            self.linear_sum = {}
-            self.squared_sum = {}
+            self.linear_sum = defaultdict(float)
+            self.squared_sum = defaultdict(float)
             for key in x.keys():
                 self.linear_sum[key] = x[key] * sample_weight
                 self.squared_sum[key] = x[key] * x[key] * sample_weight
```

Comment:
In the future we should consider using components from `stats`.

Reply:
Indeed, this will be addressed in a separate PR as it might be possible to replicate for other methods as well.

Reply:
Cool!